src/Support/Testing/Fakes/KafkaFake.php

```diff
@@ -30,6 +34,52 @@ public function publishOn(string $topic, ?string $broker = null): ProducerBuilde
         return $this->makeProducerBuilderFake($topic, $broker);
     }
 
+    /**
+     * Return a ConsumerBuilder instance.
+     *
+     * @param array $topics
+     * @param string|null $groupId
+     * @param string|null $brokers
+     * @return \Junges\Kafka\Support\Testing\Fakes\ConsumerBuilderFake
+     */
+    public function createConsumer(array $topics = [], string $groupId = null, string $brokers = null): ConsumerBuilderFake
+    {
+        return ConsumerBuilderFake::create(
+            brokers: $brokers ?? config('kafka.brokers'),
+            topics: $topics,
+            groupId: $groupId ?? config('kafka.consumer_group_id')
+        )->setMessages(
+            $this->messagesToConsume
+        );
+    }
+
+    /**
+     * Add a mock messages to consume.
+     *
+     * @param \Junges\Kafka\Contracts\KafkaConsumerMessage|Junges\Kafka\Contracts\KafkaConsumerMessage[] $messages
+     * @return void
+     */
+    public function addMockConsumerMessages(KafkaConsumerMessage|array $messages): void
```

Comment:
Maybe we can use `shouldReceive` or something similar, to keep consistency with laravel first party packages?

Reply:
Do you mean return an instance of Mockery?

Reply:
No, I meant renaning the method to `shouldReceive`, or `shouldReceiveMessages`, something similar. Just a suggestion.